teamA/leeHyeonTae/controllers/userController.js

```diff
@@ -0,0 +1,23 @@
+const userService = require('../services/userService');
+
+const signUp = async (req, res) => {
+    try{
+        const { name, email, profile_image, password } = req.body;
+
+        if( !name || !email || !profile_image || !password ){
+            return res.status(400).json({ message : 'error! key_error of userController'});
+        }
+
+        await userService.signUp( name, email, profile_image, password);
+
+        return res.status(201).json({ message: 'user signup complete! '});
```

Comment:
westudy 과제를 확인하여 성공 시 반환하는 메시지를 수정해주세요!
```suggestion
        return res.status(201).json({ message: 'userCreated'});
```

Reply:
유저의 데이터가 잘 db에 저장되었는지 여부를 테이블 스크린샷을 찍어서 PR에 같이 첨부하여 올려주세요.